### ISSUE TITLE
About N-14 dataset

### ISSUE THREAD

**Issue by ChipsterLin** [circuitnet/CircuitNet#40]
Hello! I noticed that your team mentioned a new dataset in the published Circuitnet2.0. In the N14 dataset available on GitHub, I found five datasets: RISCY, RISCY-FPU, NVDLA-small, Vortex-small, and zero-riscy. However, the mentioned dataset in the article includes seven datasets: RISCY, RISCY-FPU, zero-riscy, OpenC910-1, Vortex-small, Vortex-large, and NVDLA-large. I have a few questions: It appears that OpenC910-1, NVDLA-large, and Vortex-large are missing, while NVDLA-small is included. Additionally, I would like to inquire about the possibility of open-sourcing the corresponding LEF and DEF files for further research. Thank you very much!

**Comment by BatCaveDev-PG** [comment]
Yes, i have same issue. If you can close this issue for further research it would be helpful.
Thank you very much!
